tests/src/Mod/Material/App/TestMaterials.cpp

```diff
@@ -222,8 +222,8 @@ TEST_F(TestMaterial, TestAddAppearanceModel)
 
 QString parseQuantity(const char *string)
 {
```

Comment:
The signature of this function could be changed to return a std::string. This would also simplify the code of the tests below.

Reply:
Unless I'm missing something, that would also require changing Materials to use std::string. I already commented on that above. Do you want that conversion to happen as a part of this PR?

Reply:
> Unless I'm missing something, that would also require changing Materials to use std::string

This is only a function of the unit tests and doesn't affect the Material module.

> Do you want that conversion to happen as a part of this PR?

Can be done in a separate PR.

Reply:
> This is only a function of the unit tests and doesn't affect the Material module.

It is used as 
```
EXPECT_EQ(properties[QString::fromStdString("Density")]->getString(), parseQuantity("7900.00 kg/m^3"));
```
where `properties` comes from Material module and returns `QString`. Therefore compilation terminates with
```
gtest.h:1379:11: error: no match for ‘operator==’ (operand types are ‘const QString’ and ‘const std::__cxx11::basic_string<char>’)
```

Reply:
You're right! Sorry for the noise.